core/server.go

```diff
@@ -550,10 +557,14 @@ func (s *Server) NewSignatoryRequest(userOp *operation.UserOperation, solverOps
 	s.mu.Lock()
 	defer s.mu.Unlock()
 
-	// Retrieve dApp signatories
-	signatories, relayErr := s.getDAppSignatories(userOp.Control)
-	if relayErr != nil {
-		log.Info("failed to get dApp signatories", "control", userOp.Control.Hex(), "err", relayErr.Message)
+	if signatories == nil {
+		var relayErr *relayerror.Error
+		signatories, relayErr = s.getDAppSignatories(userOp.Control)
+		if relayErr != nil {
+			log.Info("failed to get dApp signatories", "control", userOp.Control.Hex(), "err", relayErr.Message)
+			registerBundleError(userOpHash, ErrSignatoryOffline)
```

Comment:
Is this error appropriate here?

Reply:
changed in https://github.com/FastLane-Labs/atlas-operations-relay/pull/60/commits/e64aa0182f2ce1d243af0ed5139bacdf389eba9e